core/src/main/scala/org/apache/spark/deploy/master/Master.scala

```diff
@@ -774,7 +778,7 @@ private[spark] class Master(
       case fnf: FileNotFoundException =>
         // Event logging is enabled for this application, but no event logs are found
         val title = s"Application history not found (${app.id})"
-        var msg = s"No event logs found for application $appName in ${app.desc.eventLogDir}."
+        var msg = s"No event logs found for application $appName in ${app.desc.eventLogDir.get}."
```

Comment:
the `.get` is a minor unrelated change, right?

I think this looks good in that it changes a situation where Spark does nothing and just shows an error, into one where it shows something, which may be needed for debugging.